wire-ios/Configuration/url.json

```diff
@@ -1,5 +1,23 @@
 {
-    "wireAppOnItunes": "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8",
+    "appOnItunes": "https://geo.itunes.apple.com/us/app/wire/id930944768?mt=8",
     "support": "https://support.wire.com",
-    "randomProfilePictureSource": "https://source.unsplash.com/800x800/?landscape"
+    "searchSupport": "https://wire.com/support/username",
+    "website": "https://wire.com",
+    "emailAlreadyInUse": "https://support.wire.com/hc/articles/115004082129-My-email-address-is-already-in-use-and-I-cannot-create-an-account-What-can-I-do-",
+    "whyToVerifyFingerprintArticle": "https://support.wire.com/hc/articles/207859815-Why-should-I-verify-my-conversations",
+    "howToVerifyFingerprintArticle": "https://support.wire.com/hc/articles/207692235-How-can-I-compare-key-fingerprints-",
+    "privacyPolicy": "https://wire.com/legal",
+    "legal": "https://wire.com/legal",
+    "licenseInformation": "https://wire.com/legal/licenses/embed",
+    "passwordResetInfo": "https://account.wire.com/forgot",
+    "askSupportArticle": "https://support.wire.com/hc/requests/new",
+    "reportAbuse": "https://support.wire.com/hc/requests/new",
+    "pricingInfo": "https://wire.com/pricing",
+    "wireEnterpriseInfo": "https://wire.com/pricing",
+    "legalHoldInfo": "https://support.wire.com/hc/articles/360002018278-What-is-legal-hold-",
+    "guestLinksInfo": "https://support.wire.com/hc/articles/360000574069-Share-a-link-with-a-person-without-a-Wire-account-to-join-a-guest-room-conversation-in-my-team",
+    "unreachableBackendInfo": "https://support.wire.com/hc/articles/9357718008093-Backend",
+    "federationInfo": "https://support.wire.com/hc/categories/4719917054365-Federation",
+    "mlsInfo": "https://support.wire.com/hc/articles/12434725011485-Messaging-Layer-Security-MLS-",
+    "endToEndIdentityInfo": "https://support.wire.com/hc/articles/9211300150685-End-to-end-identity"
```

Comment:
I think it should be very clear what the purpose of each of these links is so that anyone that needs to fill it out knows what content to put. 

For example, `website` and `legal` seem self explanatory but perhaps not to everyone, or someone who doesn't have all the context. It would be better to have some additional comments:

`website`: link to the app's homepage.
`legal`: link to app's homepage for legal information, e.g terms of use and data processing addendum.
`emailAlreadyInUse`: shown when the user tries to create an account with an email that is already in use. Links to support page explaining issue and what can be done.

Reply:
added